prog/weave-npc/main.go

```diff
@@ -127,7 +128,7 @@ func createBaseRules(ipt *iptables.IPTables, ips ipset.Interface) error {
 
 	// If the destination address is not any of the local pods, let it through
 	if err := ipt.Append(npc.TableFilter, npc.MainChain,
-		"-m", "physdev", "--physdev-out=vethwe-bridge", "-j", "ACCEPT"); err != nil {
+		"-m", "physdev", "--physdev-out="+bridgeName, "-j", "ACCEPT"); err != nil {
```

Comment:
I think `bridgeName` is a bit confusing here. IMO, `bridgePortName` would be better.